projects/Switch/packages/ishiiruka/core_info/ishiiruka_libretro.info

```diff
@@ -0,0 +1,12 @@
+display_name = "Nintendo - GameCube / Wii (Ishiiruka)"
```

Comment:
It is not possible to update [core info package](https://github.com/libretro/Lakka-LibreELEC/blob/Lakka-V2.1-dev/packages/libretro/core-info/package.mk) as it is available [upstream](https://github.com/libretro/libretro-super/blob/master/dist/info/ishiiruka_libretro.info).

Reply:
My original plan was to add the core and let the users decide which one to use (Dolphin or Ishiiruka) on the Switch. However Lakka didn't let me load a .iso with Ishiiruka without a core info, hence this hacky way of adding the core info.

I can make a PR for the core info package but I wanted to wait for the core to be actually finished. I also need to make sure that the core  can load all of Dolphin's file formats.

Just tell me if you want the PR

Reply:
Sure no problem lets wait a little the core to be polished ;-)

Reply:
Well it's not finished, I don't think all the inputs are there (there is only GC, no Wii), there is no Vulkan / D3D11 support...